src/main/resources/templates/db_scripts/schema.json

```diff
@@ -392,12 +392,6 @@
           "caseSensitive": false,
           "removeAccents": true
         },
-        {
-          "fieldName": "classifications",
-          "tOps": "ADD",
```

Comment:
I beleive, it will not be enough just to remove defenition of gin index. It will not be deleted if it already exists. So "tOps" value should be changed to "DELETE" for completly removing unnecessary gin index

Reply:
@julianladisch @adamdickmeiss  Please could this be confirmed, does RAML Module Builder need this to be left in and marked as delete rather than removed entirely?

Reply:
https://github.com/folio-org/raml-module-builder#removing-an-index
> When upgrading a module via the Tenant API, an index is deleted if either "tOps": "DELETE" is set or the complete index entry is removed. Note that indexes are the only elements where removing the entry in schema.json removes them from the database.